### PR TITLE
[MRG+1] Coreg-GUI: Error message for FIFF file without cardinal points

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Display a proper error message in cases like #4589

### CODE REVIEW

File: mne/gui/_file_traits.py [hunk 1]
```diff
@@ -323,6 +320,20 @@ def _get_inst(self):
...
+                has_point = {FIFF.FIFFV_POINT_LPA: False,
+                             FIFF.FIFFV_POINT_NASION: False,
+                             FIFF.FIFFV_POINT_RPA: False}
+                for d in info['dig']:
+                    if d['kind'] == FIFF.FIFFV_POINT_CARDINAL:
+                        has_point[d['ident']] = True
+                if not all(has_point.values()):
+                    error(None, "The selected FIFF file does not contain "
+                          "cardinal points. Please select a different file.",
+                          "Error Reading FIFF File")
```
Comment (agramfort) [inline mne/gui/_file_traits.py]: can you say which ones are missing?

File: mne/gui/_file_traits.py [hunk 2]
```diff
@@ -338,72 +349,49 @@ def _get_inst_fname(self):
             return '-'
 
     @cached_property
-    def _get_inst_points(self):
-        if not self.inst:
+    def _get__hsp_points(self):
```
Comment (agramfort) [inline mne/gui/_file_traits.py]: why 2 _ ?

Comment (christianbrodbeck) [reply mne/gui/_file_traits.py]: Because `_hsp_pionts` is private – `_get_` + `_hsp_points`

Comment (christianbrodbeck) [reply mne/gui/_file_traits.py]: (It's a method name that the HasPrivateTraits superclass recognizes)

**Comment by larsoner** [comment]
Thanks @christianbrodbeck

**Pull request merged**